lib/api/configs.js

```diff
@@ -0,0 +1,78 @@
+var linz = require('../../'),
+    async = require('async'),
+    clone = require('clone');
+
+/**
+ * Get linz config by reference or by cloning a copy
+ * @param  {String} configName      Name of config
+ * @param  {Boolean} copy           Specify if result should be a cloned copy or by reference. Default to return a reference.
+ * @return {Object}                 Config object either by a cloned copy or by reference
+ */
+function get (configName, copy) {
+
+    if (copy) {
+        return clone(linz.get('configs')[configName]);
+    }
+
+    return linz.get('configs')[configName];
+}
+
+
+/**
+ * Get config permissions
+ * @param {Object}   req      Request data
+ * @param {String}   configName Config name
+ * @param {Function} cb        Callback function
+ * @return {Object}
+ */
+function getPermissions(req, configName, cb) {
+
+    if (!linz.get('disable config cache') && !req.linz.cache.permissions.configs.invalidate && req.session.configs && req.session.configs[configName]) {
+        return cb(null, req.session.configs[configName]);
+    }
+
```

Comment:
Could we please have some debug output here saying `Priming config cache.` The debug should use the `debug` module, with a setting of `linz:cache`. That will allow us to get a handle of when the cache is/isn't being regenerated.